ts-apps/decision-logic-visualizer/src/lib/displayers/flow/sf-custom-nodes/ubool-var.svelte

```diff
@@ -39,8 +41,9 @@ TODO: Look into why this is the case --- are they not re-mounting the ubool-var
           ).getUnique(data.context)
           l4Conn.inlineExprs([uniq], ladderEnv.getVersionedTextDocIdentifier())
         }}
+        aria-label="Expand inline expression"
```

Comment:
Oh wait one more thing: Call it 'inline expression'

Reply:
Perhaps think a bit about how best to put it?

Reply:
I agree it’s important to clarify what’s being expanded. I’ve gone with "Expand nested clause" to keep it concise and communicative for screen readers, while matching how legal users understand the interface. Rather than spelling out “with its definition,” which risks being overly verbose or ambiguous, this phrasing conveys both the structure (a nested clause) and the action (expansion) in a way that aligns with UI/UX and accessibility conventions.